src/java.base/share/classes/jdk/internal/foreign/abi/aarch64/CallArranger.java

```diff
@@ -77,6 +79,11 @@ public abstract class CallArranger {
     public static final int MAX_REGISTER_ARGUMENTS = 8;
 
     private static final VMStorage INDIRECT_RESULT = r8;
+    private static final boolean isWinOS;
+
+    static {
+        isWinOS = GetPropertyAction.privilegedGetProperty("os.name").startsWith("Windows");
+    }
```

Comment:
Please insert a blank line to separate this from the unrelated field `INDIRECT_RESULT` and collapse it to:
```java
    private static final boolean isWinOS = GetPropertyAction.privilegedGetProperty("os.name").startsWith("Windows");
```

Reply:
Fixed as suggested above.